runtime/darwinia-parachain/src/pallets/remote_governance.rs

```diff
@@ -0,0 +1,46 @@
+// --- paritytech ---
+use frame_support::traits::{EnsureOrigin, Get};
+use frame_system::RawOrigin;
+// --- darwinia-network ---
+use crate::*;
+use dp_common_runtime::remote_governance::Config;
+
+pub struct EnsureSpecific;
+impl<O: Into<Result<RawOrigin<AccountId>, O>> + From<RawOrigin<AccountId>>> EnsureOrigin<O>
+	for EnsureSpecific
+{
+	type Success = AccountId;
+
+	fn try_origin(o: O) -> Result<Self::Success, O> {
+		o.into().and_then(|o| match o {
+			RawOrigin::Signed(who)
+				if who
+					== array_bytes::hex_n_into_unchecked(
+						"0xc778fc2665f3f6ee9623594e5d1fab9dbd557149542c5edacbcc543a82c9d780",
```

Comment:
How this derive account generated?

Is chain id included in derive process?

Reply:
Previous multi-sig root key.

Reply:
"multi-sig root key" looks not correct, this should be the LCMP derived key from Darwinia's Root?

Reply:
It looks the same as Crab's relevant key: https://github.com/darwinia-network/darwinia-parachain/blob/315888f9f024004ab54cad67f4ce6321cda6ae3e/runtime/crab-parachain/src/pallets/remote_governance.rs#L19

Reply:
```
pub fn derive_account_id<AccountId>(bridge_id: ChainId, id: SourceAccount<AccountId>) -> H256
where
	AccountId: Encode,
{
	match id {
		SourceAccount::Root =>
			(ROOT_ACCOUNT_DERIVATION_PREFIX, bridge_id).using_encoded(blake2_256),
		SourceAccount::Account(id) =>
			(ACCOUNT_DERIVATION_PREFIX, bridge_id, id).using_encoded(blake2_256),
	}
	.into()
}
```

This is the derive algorithm, different chains for same origin(ROOT) will result different derived account.

Please add comments here too.

Reply:
![image](https://user-images.githubusercontent.com/16152305/199467010-54a4701f-3525-4f06-a821-df3aabdc20f7.png)
```
λ subalfred key 5GaFKsnKnc2gKQwrCBJzfMxBrS2A7rjrLC4qPynJzav3SJCu
public-key 0xc778fc2665f3f6ee9623594e5d1fab9dbd557149542c5edacbcc543a82c9d780
Substrate 5GaFKsnKnc2gKQwrCBJzfMxBrS2A7rjrLC4qPynJzav3SJCu
```

Reply:
It's multi-sig root key.

Reply:
I see, my mistake, I though it was:

https://github.com/darwinia-network/darwinia-parachain/blob/main/runtime/common/src/remote_governance/mod.rs#L201-L206